### PR TITLE
Update: ListPicker empty state

### PR DESCRIPTION
**Pull request by vinteo**
image

### CODE REVIEW

File: test/components/adslotUi/ListPickerPureComponentTest.js [hunk 1]
```diff
@@ -28,6 +28,11 @@ describe('ListPickerPureComponent', () => {
 
     const gridElement = component.props.children;
     expect(gridElement.type).to.equal((<Grid />).type);
+
+    const emptyElement = gridElement.props.children[2];
+    expect(emptyElement.type).to.equal((<Empty />).type);
+    expect(emptyElement.props.collection).to.deep.equal([]);
```
Comment (camjc) [inline test/components/adslotUi/ListPickerPureComponentTest.js]: to have length 0?

**Comment by camjc** [comment]
LGTM once we have a solution for the bottom border

**Comment by vinteo** [comment]
@camjc merge?

**Comment by camjc** [comment]
Merging!

**Pull request merged**